### ISSUE TITLE
Add a "decline benefits" option to GitHub Sponsors

### ISSUE THREAD

**Issue by juliaferraioli** [todogroup/gh-issues#73]
Just like when donating to nonprofits, there should be a "please do not publicize my sponsorship" option for GitHub sponsors.

Or even better, **explicit consent** should be built into all sponsorship tiers so that project maintainers do not have to create a custom tier for this.

**Comment by ljharb** [comment]
Sponsorships can already be private.
<img width="588" alt="Screen Shot 2022-06-29 at 10 55 54 AM" src="https://user-images.githubusercontent.com/45469/176503921-dfa7bebd-15db-40e1-a6da-871b966468ad.png">

**Comment by juliaferraioli** [comment]
That's about whether the UI shows that you're a sponsor. Sponsorship tiers often have some sort of "recognition" tied to them, and there's no way to decline that, leading to sponsors having to figure out how to work around that.

For instance, here are two sponsorship tiers on a ✨randomly selected✨ profile:

![Screen Shot 2022-06-29 at 11 03 39 AM](https://user-images.githubusercontent.com/281597/176505538-9dd0926f-beb1-4add-913f-bd1d08b72255.png)

If you select the higher tier I can't say, "please don't add me to the README".

**Comment by ljharb** [comment]
That's true, but the sponsor will receive an email saying it's a private sponsorship. Is the ask to hide the sponsor's identity *from* the sponsor?

**Comment by juliaferraioli** [comment]
No, that's not what I'm asking for. I'm asking for a way for people to sponsor at a tier, but decline public recognition.

So if I want to sponsor at $25/month, I want to be able to decline the benefits/awards.

**Comment by ljharb** [comment]
Gotcha, thanks for clarifying!

**Comment by juliaferraioli** [comment]
No problem! These things are nuanced and i wrote this probably too quickly :-)
